### ISSUE TITLE
[Feature Request] Being able to set smaa,fxaa,dls etc for vkbasalt

### ISSUE THREAD

**Issue by Toadfield** [benjamimgois/goverlay#78]
I know that I can set cas sharpening in goverlay,but that's not all that vkbasalt offers.
I also want that I can change all settings for smaa and fxaa,like how smaa detects edges or how much search steps it performs etc.,basically all the things that I can set manually in the vkbasalt config.
I know that I can use smaa and fxaa with reshade,but I can't change all the settings for it like with vkbasalt,if I'm not wrong.

**Comment by benjamimgois** [comment]
You're right... i didn't implement those advanced functions since i didn't thought lots of users would make use of then. You're the first one to actually mention it, Right now it's not a priority but i can come back to it in the future.

**Comment by Toadfield** [comment]
ok

**Comment by blackCauldron7** [comment]
Please implement this.
